08_deploy_bmo.sh

```diff
@@ -12,10 +12,36 @@ export BMOPATH="$GOPATH/src/github.com/metal3-io/baremetal-operator"
 # Make a local copy of the baremetal-operator code to make changes
 cp -r $BMOPATH/deploy ocp/.
 sed -i 's/namespace: .*/namespace: openshift-machine-api/g' ocp/deploy/role_binding.yaml
+cp $SCRIPTDIR/operator_ironic.yaml ocp/deploy
+cp $SCRIPTDIR/ironic_bmo_configmap.yaml ocp/deploy
+sed -i "s#__RHCOS_IMAGE_URL__#${RHCOS_IMAGE_URL}#" ocp/deploy/ironic_bmo_configmap.yaml
+
+# Kill the dnsmasq container on the host since it is performing DHCP and doesn't
+# allow our pod in openshift to take over.
+for name in dnsmasq ironic-inspector ; do
```

Comment:
What about the rest of the ironic containers?

Reply:
If we kill all the containers we lose the ironic database and cleanup will not work.  eg 'make clean' in dev-scripts will fail to remove the ironic managed masters.

Reply:
OK, expanding the comment would help.  I imagine all of this is going away shortly anyway once the bootstrap VM ironic work lands.

Reply:
Yeah that's correct, but then we'll also need to modify make clean to not rely on the terraform cleanup (until we figure out how to reimplement destroy)